mod-source-record-storage-server/src/main/java/org/folio/services/util/AdditionalFieldsUtil.java

```diff
@@ -294,21 +294,31 @@ && isNotEmpty(((ControlField) field).getData())
    *
    * @param recordInstancePair pair of related instance and record
```

Comment:
@Aliaksandr-Fedasiuk  Please add documentation for the brand new param "isUpdateOption".

Reply:
Ready